kokoro/windows/build.bat

```diff
@@ -39,32 +39,32 @@ wget -q https://dl.google.com/android/repository/android-ndk-r21d-windows-x86_64
 unzip -q android-ndk-r21d-windows-x86_64.zip
 set ANDROID_NDK_HOME=%CD%\android-ndk-r21d
 
-REM Install WiX Toolset.
-wget -q https://github.com/wixtoolset/wix3/releases/download/wix311rtm/wix311-binaries.zip
-unzip -q -d wix wix311-binaries.zip
-set WIX=%cd%\wix
-
-REM Manually install only the required MSYS packages. Do NOT do a
-REM system update (pacman -Syu) because it is a moving target. In
-REM particular, the pacman installed on default Kokoro VMs is old, and
-REM supports only ".pkg.tar.xz" package archives, it does not support
-REM the more recent ".pkg.tar.zst" packages archives.
-c:\tools\msys64\usr\bin\bash --login -c "pacman -R --noconfirm catgets libcatgets"
-REM Use an old version of patch known to work with the msys runtime
-REM version that comes on Kokoro. Temporarily getting it from an
-REM alternate mirror, since they msys host no longer carries this
-REM version.
-wget -q http://ftp.oregonstate.edu/pub/xbmc/build-deps/win32/msys2/repos/msys2/x86_64/patch-2.7.5-1-x86_64.pkg.tar.xz
-c:\tools\msys64\usr\bin\bash --login -c "pacman -v -U --noconfirm  /t/src/patch-2.7.5-1-x86_64.pkg.tar.xz"
-wget -q http://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-binutils-2.33.1-1-any.pkg.tar.xz
-c:\tools\msys64\usr\bin\bash --login -c "pacman -U --noconfirm /t/src/mingw-w64-x86_64-binutils-2.33.1-1-any.pkg.tar.xz"
-wget -q http://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-gcc-9.2.0-2-any.pkg.tar.xz
-wget -q http://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-gcc-libs-9.2.0-2-any.pkg.tar.xz
-c:\tools\msys64\usr\bin\bash --login -c "pacman -U --noconfirm /t/src/mingw-w64-x86_64-gcc-9.2.0-2-any.pkg.tar.xz /t/src/mingw-w64-x86_64-gcc-libs-9.2.0-2-any.pkg.tar.xz"
-wget -q http://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-crt-git-8.0.0.5647.1fe2e62e-1-any.pkg.tar.xz
-c:\tools\msys64\usr\bin\bash --login -c "pacman -U --noconfirm /t/src/mingw-w64-x86_64-crt-git-8.0.0.5647.1fe2e62e-1-any.pkg.tar.xz"
-set PATH=c:\tools\msys64\mingw64\bin;c:\tools\msys64\usr\bin;%PATH%
-set BAZEL_SH=c:\tools\msys64\usr\bin\bash
+REM Download and install MSYS2, because the pre-installed version is too old.
+REM Do NOT do a system update (pacman -Syu) because it is a moving target.
+wget -q https://github.com/msys2/msys2-installer/releases/download/2020-11-09/msys2-base-x86_64-20201109.sfx.exe
+.\msys2-base-x86_64-20201109.sfx.exe -y -o%BUILD_ROOT%\
+
+REM Start empty shell to initialize MSYS2.
+%BUILD_ROOT%\msys64\usr\bin\bash -lc " "
```

Comment:
keep it as `--login -c`. For most readers `--login` is easier to understand than `-l`.

Reply:
Sure, done